tests/scripts/custodian/nlb.yaml

```diff
@@ -0,0 +1,53 @@
+policies:
+- name: delete-nlbs
+  resource: app-elb
+  filters:
+    - and:
+      - or: 
+        - type: value
+          key: LoadBalancerArn
+          op: regex
+          value: "^((?!USERKEYS).)*$"
+        - 'tag:deletesAtMidnight': present
+      - not:
+        - 'tag:doNotDelete': present
+        - 'tag:DoNotDelete': present
+        - 'tag:DeletesOnFriday': present
+        - type: value
+          key: tag:Name
+          op: regex
+          value:  "^.*DONOTDELETEKEYS.*$"
+  actions:
+    - delete
+    ### option for notifying slack / via email (needs lambda permissions)
```

Comment:
Are we intending on leaving this in as a commented section?

Reply:
good question; @bmdepesa requested a notifier for this, but we would need to do the work to setup the lambda permissions and slack integration (or email) approval. Lets get his feedback before uncommenting / removing.

Reply:
after further investigation, it was difficult for me to get a notify action to work properly.